ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -534,6 +537,7 @@ try:
         )
 
         uploader.set_region(region)
+        uploader.prepare_aws_for_upload()
```

Comment:
I was trying to call this method in the initializer, but that method depends on the region. And calling self.set_region from [here](https://github.com/SUSE/Enceladus/blob/master/ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py#L100) doesn't seem to work.

If there is a way to make those to calls in EC2ImageUploader, I would prefer that.

Reply:
You could hand over the region and just drop the set_region call, no?
As ec2uploadimg calls super and the region parameter gets set in __init__:
https://github.com/SUSE/Enceladus/blob/master/ec2utils/ec2utilsbase/lib/ec2utils/ec2utils.py#L30

Reply:
That works :+1:

Reply:
I'm ok with making region part of the constructor arguments and dropping the call.